js/web/lib/onnxjs/ort-schema/README.md

```diff
@@ -0,0 +1,11 @@
+# ORT Format File
```

Comment:
js/web/README.md must have a link to this file?

Reply:
do you suggest to add a section to reference this ?

Reply:
will add schema generation docs to /js/README.md in later changes